impl/maven-impl/src/main/java/org/apache/maven/impl/model/DefaultModelValidator.java

```diff
@@ -2251,23 +2251,20 @@ public static SourceHint gav(String gav) {
 
         public static SourceHint dependencyManagementKey(Dependency dependency) {
             String hint;
-            if (dependency.getClassifier() == null
-                    || dependency.getClassifier().trim().isEmpty()) {
-                hint = String.format(
-                        "groupId=%s, artifactId=%s, type=%s",
-                        nvl(dependency.getGroupId()), nvl(dependency.getArtifactId()), nvl(dependency.getType()));
+            if (dependency.getClassifier() == null || dependency.getClassifier().isBlank()) {
+                hint = "groupId=" + valueToValueString(dependency.getGroupId())
```

Comment:
so this can really have groupId= rather than groupId="" ?
That seems strange

Reply:
> so this can really have groupId= rather than groupId="" ? 

according to the original codes, yes.

> That seems strange

if you ask me for my ideas, yes, it is strange.